apps/admin-ui/public/resources/en/realm-settings.json

```
@@ -114,17 +114,20 @@
   "loginWithEmailHelpText": "Allow users to log in with their email address.",
   "duplicateEmailsAllowed": "Duplicate emails",
   "duplicateEmailsHelpText": "Allow multiple users to have the same email address. Changing this setting will also clear the user's cache. It is recommended to manually update email constraints of existing users in the database after switching off support for duplicate email addresses.",
-  "provideEmailTitle": "Provide your email address",
-  "provideEmail": "To test connection, you should provide your email address first.",
   "verifyEmail": "Verify email",
   "verifyEmailHelpText": "Require user to verify their email address after initial login or after address changes are submitted.",
   "userInfoSettings": "User info settings",
   "editUsernameAllowed": "Edit username",
   "enableSwitchSuccess": "{{switch}} changed successfully",
   "enableSwitchError": "Could not enable / disable due to {{error}}",
   "testConnection": "Test connection",
+  "testConnectionHint": {
+    "withEmail": "When testing the connection an e-mail will be sent to the currently logged-in user ({{email}}).",
+    "withoutEmail": "To test the connection you must first configure an e-mail address for your user.",
```

@jonkoops Let's make the `withoutEmail` as clear as `withEmail`.  Suggested change:
```suggestion
    "withoutEmail": "To test the connection you must configure an e-mail address for the current user ({{userName}}).",
```

---

Done, I've also changed `withEmail` to be more consistent with the flow of `withoutEmail`.